src/components/Register.tsx

```diff
@@ -32,32 +67,124 @@ const Register: React.FC = () => {
     })
   }
 
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target
+    updateNewUser({ ...newUser, [name]: value })
+  }
+
+  const [createUserMutation] = useCreateUserMutation({
+    update(cache, { data: createdUser }) {
+      if (!createdUser) return
+      cache.modify({
+        fields: {
+          users(existingUserRefs: UsersConnection, { readField }) {
+            const { user } = createdUser.createUser as CreateUserPayload
+            if (!user) return existingUserRefs
+
+            const newUserRef = cache.writeFragment({
+              data: user,
+              fragment: addNewUser,
+            })
+
+            // Quick safety check - if the new user is already
+            // present in the cache, we don't need to add it again.
+            if (
+              existingUserRefs.nodes.some((ref) => (ref ? readField('id', ref) === user.id : false))
+            ) {
+              return existingUserRefs
+            }
+
+            return [...existingUserRefs.nodes, newUserRef]
+          },
+        },
+      })
+    },
+    onCompleted: () => {
+      changeSnackback(submitedObject)
+      setTimeout(() => {
+        removeSnackbar()
+      }, 2000)
+    },
+  })
+
+  const updateUser = async () => {
+    try {
+      const { username, password, email, role } = newUser
+      const updatedUser = await createUserMutation({
```

Comment:
`updatedUser` is not used anywhere. Do you need to declare it at all?

Why not just `await createUserMutation({...` ?

Reply:
Yes. I think I was just doing something else and left it there. Thanks.